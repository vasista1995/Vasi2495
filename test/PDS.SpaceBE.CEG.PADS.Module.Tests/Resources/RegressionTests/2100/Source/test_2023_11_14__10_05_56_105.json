{
  "IdSource": "SPACEACT3:CEG:BE:81771705:62mm:system soldering:K2-T-Loettemp:101007:332713727:0",
  "Data_flat_Metadata": {
    "Plant": "0600",
    "Line": "62mm",
    "ProductionOrder": "81771705",
    "Batch": "UZ38277P0",
    "MaterialNumber": "42814",
    "ProductGroup": "M-62MM3.GEN",
    "MeasurementRecipe": "1",
    "ProcessEquipment": "10000973",
    "OrderType": "2100",
    "QrkGroup": "2100",
    "ProcessGroup": "system_soldering",
    "SpcClass": "2100",
    "ParameterType": "2100",
    "LongParameterName": "K2-Löttemp",
    "MaterialNumberText": "2100",
    "ProcessEquipmentName": "VADU10",
    "Operator": "2100",
    "ParameterClass": "NON_PPAP",
    "Segment": "HPS",
    "Package": "62mm",
    "Module": "11 substrate-baseplate interconnect",
    "PosName": "2100",
    "FourDReport": "*",
    "ParameterName": "K2-T-Loettemp",
    "ParameterUnit": "°C",
    "ParameterOper": "system soldering",
    "SiteKey": "CEG",
    "ChannelId": "101007",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-27T10:04:18Z",
    "CreatedTimestamp": "2023-10-27T12:04:18Z",
    "UpdatedTimestamp": "2023-10-27T12:04:18Z",
    "UpdatedTimestampUtc": "2023-10-27T10:04:18Z",
    "ExportedTimestampUtc": "2023-10-27T11:58:31Z",
    "ExportedTimestamp": "2023-10-27T11:58:31Z",
    "SampleTimestampUtc": "2023-10-27T09:58:31Z",
    "SampleTimestamp": "2023-10-27T11:58:31Z",
    "SourceDataLevel": "C",
    "Equipment": "VADU10",
    "ChannelName": "K2-T-Loettemp/Recipe1/VADU>5",
    "RvStoreFlag": "Y",
    "CalcCreated": "07/25/2023 18:16:10",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 9920560,
    "ViolCount": 0,
    "CalculationStrategy": "ND 4.5 Sigma",
    "LdsID": "2100",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 290.0,
        "SpecLow": 285.0,
        "SpecTarget": 287.5,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNNNNYNNYYYYYYYYYYNNNYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "287,7",
        "Median": "287,7",
        "Sigma": "0",
        "Min": "287,7",
        "Max": "287,7",
        "Q1": "287,7",
        "Q3": "287,7"
      },
      "ControlLimits": {
        "ctrl_mean_low": 286.693453271218,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 287.754632857143,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 288.815812443067,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 287.688571428572,
        "ExtEwmaMV": 287.676633459978,
        "ExtMsMV": 0.148492424049161,
        "EwmaS": 0.0,
        "EwmaR": 0.0,
        "ExtEwmaMVLCL": 287.400896405133,
        "ExtEwmaMVCenter": 287.754632857143,
        "ExtEwmaMVUCL": 288.108369309154,
        "ExtMaMVLCL": 287.487233233807,
        "ExtMaMVCenter": 287.754632857143,
        "ExtMaMVUCL": 288.02203248048,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.000389638210646436,
        "ExtMSMVCenter": 0.183741681936864,
        "ExtMSMVUCL": 0.738097452050226,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 286.693453271218,
        "RawUal": 288.815812443067,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 332713727,
        "Seqnr": 1,
        "Value": 287.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 1
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CEG:BE:62mm::K2-T-Loettemp:101007",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "HPSModFAUF:81771705",
    "IdSystemName": "HPSModFAUF",
    "Type": "Fauf"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CEG:BE:81771705:62mm:system soldering:K2-T-Loettemp:101007:332713727:0"
  }
}