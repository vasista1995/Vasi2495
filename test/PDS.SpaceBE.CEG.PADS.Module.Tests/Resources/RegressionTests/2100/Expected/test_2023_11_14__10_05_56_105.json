{
  "Data_flat_Metadata": {
    "Line": "62mm",
    "ProductionOrder": "81771705",
    "Batch": "UZ38277P0",
    "MaterialNumber": "42814",
    "ProductGroup": "M-62MM3.GEN",
    "MeasurementRecipe": "1",
    "OrderType": "2100",
    "QrkGroup": "2100",
    "ProcessGroup": "system_soldering",
    "SpcClass": "2100",
    "CustomComment": "system soldering",
    "Segment": "HPS",
    "MeasLot": "81771705",
    "ParameterOper": "system soldering",
    "SiteKey": "CEG",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-27T09:58:31Z",
    "BeginTimestamp": "2023-10-27T11:58:31Z",
    "EndTimestampUtc": "2023-10-27T09:58:31Z",
    "EndTimestamp": "2023-10-27T11:58:31Z",
    "Equipment": "VADU10"
  },
  "Data_flat_StrangeMetadata": {
    "MaterialNumberText": "2100",
    "Operator": "2100",
    "ParameterClass": "NON_PPAP",
    "Package": "62mm",
    "Module": "11 substrate-baseplate interconnect",
    "PosName": "2100"
  },
  "Data_1list_Parameters": [
    {
      "process_equipment": "10000973",
      "ProcessEquipmentName": "VADU10",
      "LongParameterName": "K2-Löttemp",
      "Data_1list_RawValues": [
        {
          "process_equipment": "10000973",
          "MeasurementEquipment": "VADU10",
          "ProcessEquipmentName": "VADU10",
          "Seqnr": 1,
          "SampleId": 332713727,
          "Value": 287.7,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-27T11:58:31Z",
          "SampleTimestampUtc": "2023-10-27T09:58:31Z",
          "CreatedTimestamp": "2023-10-27T12:04:18Z",
          "CreatedTimestampUtc": "2023-10-27T10:04:18Z",
          "UpdatedTimestamp": "2023-10-27T12:04:18Z",
          "UpdatedTimestampUtc": "2023-10-27T10:04:18Z",
          "NumViolations": 0,
          "SampleMean": 287.7,
          "SampleMin": 287.7,
          "SampleMax": 287.7,
          "SampleMedian": 287.7,
          "SampleStdev": 0.0,
          "SampleQ1": 287.7,
          "SampleQ3": 287.7,
          "SampleSize": 1
        }
      ],
      "ParameterType": "2100",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 290.0,
          "spec_low": 285.0,
          "target": 287.5
        },
        "ControlLimits": {
          "ctrl_mean_low": 286.693453271218,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 287.754632857143,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 288.815812443067,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "CtrlRangeHighEnabled": "N"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 1,
        "base_count": 1,
        "flagged_count": 0,
        "mean": 287.7,
        "median": 287.7,
        "stdev": 0.0,
        "range": 0.0,
        "min": 287.7,
        "max": 287.7,
        "q02": 287.7,
        "q05": 287.7,
        "q25": 287.7,
        "q75": 287.7,
        "q95": 287.7,
        "q98": 287.7,
        "NumViolations": 0,
        "Samples": "332713727"
      },
      "ParameterName": "K2-T-Loettemp",
      "ParameterUnit": "°C",
      "ChannelId": "101007",
      "SourceDataLevel": "C",
      "channel_name": "K2-T-Loettemp Recipe1 VADU>5",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-27T10:04:18Z",
      "created_timestamp": "2023-10-27T12:04:18Z",
      "updated_timestamp_utc": "2023-10-27T10:04:18Z",
      "updated_timestamp": "2023-10-27T12:04:18Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-10-27T10:04:18Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED3:CEG:BE:62mm:system soldering:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "HPSModFAUF:81771705",
    "IdSystemName": "HPSModFAUF",
    "Type": "Fauf"
  },
  "SystemLog": {
    "UniqueSourceKey": "CEG:BE:81771705:62mm:system soldering:K2-T-Loettemp:101007:332713727:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-27T11:58:31Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "HPSModFAUF:81771705:SPACEAGGED3:CEG:BE:62mm:system soldering::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "CEG"
  }
}